components/BarcodeScanner.vue

```diff
@@ -0,0 +1,334 @@
+<template>
+  <div
+    ref="videoContainer"
+    class="videoContainer row justify-center items-center">
+    <!-- Spinners -->
+    <div class="full-height row items-center">
+      <slot
+        v-if="loadingCamera"
+        name="cameraSpinner">
+        <Spinner />
+      </slot>
+      <slot
+        v-if="scanning"
+        name="scannerSpinner">
+        <Spinner />
+      </slot>
+    </div>
+    <!-- Video Stream -->
+    <video
+      class="dbrScanner-video"
+      playsinline="true" />
+    <!-- Guide -->
+    <canvas
+      v-show="!loadingCamera && !scanning"
+      id="guide"
+      style="position: absolute; top:0; height: 100%; width: 100%" />
+    <!-- Tip Text -->
+    <div
+      v-if="!loadingCamera && !scanning"
+      class="text-white"
+      style="position: absolute;"
+      :style="`bottom: calc(${regionTop}% - 30px)`">
+      {{tipText}}
+    </div>
+    <!-- Close -->
+    <q-btn
+      v-if="!loadingCamera && !scanning"
+      flat
+      fab
+      :ripple="false"
+      size="16px"
+      color="white"
+      icon="fas fa-times"
+      class="q-ma-sm close-btn"
+      @click.native="close()" />
+    <!-- Buttons -->
+    <Buttons
+      :camera-list="cameraList"
+      @updateCamera="updateCamera($event)"
+      @upload="upload($event)" />
+  </div>
+</template>
+
+<script>
+/*!
+  * Copyright (c) 2022 Digital Bazaar, Inc. All rights reserved.
+  */
+import DBR from '../helpers/dbr.js';
+import driverLicenseFields from '../helpers/driverLicenseFields.js';
+import Buttons from './Buttons.vue';
+import Spinner from './Spinner.vue';
+
+DBR.BarcodeReader.license = '100902848';
```

Comment:
This should be provided by the top level application as a parameter.

Reply:
@dlongley does this mean from `bedrock.web.config`?

Reply:
Yeah, that's probably how it should be done.